backend/dataall/core/permissions/api/queries.py

```diff
@@ -16,3 +19,9 @@
     type=gql.Ref('GroupSearchResult'),
     resolver=list_tenant_groups,
 )
+
```

Comment:
I think it is already in the actual service, but this API should be part of the organization module

Reply:
As I wrote above, I think, the permission related API should stay in permission folder.